hydromt/data_adapter/dataframe.py

```diff
@@ -35,6 +35,7 @@ def __init__(
         unit_add: dict = {},
         meta: dict = {},
         attrs: dict = {},
+        extent=None,
```

Comment:
We do actually, because in catalog we call `adapter(...)` so if this keyword is not known it will either throw an error, or pass it down as a kwarg. So we need it here, but I agree that it should be made more clear that nothing is done with it here

Reply:
As long as DataFrame sources do not define the extent in their catalog it will be passed to the DataFrameAdapter right? If think it's better to throw a meaningful error when parsing the data catalogs rather than masking it.

Reply:
I did not consider that, you are right! Removed the keyword from Dataframe :)